docs/faq.rst

```diff
@@ -64,6 +64,15 @@ environment, and will be set to ``True`` when building on RTD::
     Woo
     {% endif %}
 
+My project requires different settings than that are available under Admin
+--------------------------------------------------------------------------
+
+Read the Docs offers additional flag settings. To enable these settings, please open an
```

Comment:
a request issue

Reply:
This was a very silly mistake from my part.
Thank you.
I have made the changes.

Reply:
No silly, no dumb, no stupid, no nothing. Just keep rocking with your PRs and helping us to build the best documentation platform ;) Thanks! :heart:

Reply:
@humitos 
Thank you very much.
This was very motivating for me. :smile:

Reply:
For the record, I agree with @humitos. :) Typos sneak in everywhere.

Reply:
I do agree on this point. :smile: